src/styles/BookingPage.styled.js

```diff
@@ -0,0 +1,149 @@
+import styled from "styled-components";
+
+export const BookingPageStyle = styled.div`
+  /* @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&family=Rubik:wght@500;600&display=swap'); */
+
+  display: flex;
+position: relative;
+  margin: auto;
+  width: max-content;
+  flex-direction: column;
+
+  button {
+    margin: auto;
+    margin-top: 100px;
+    width: 190px;
+    height: 43px;
+    background-color: #352c9a;
+    font-family: "Rubik", sans-serif;
+    font-size: 16px;
+    font-weight: 600;
+    color: white;
+    cursor: pointer;
+
+    &:active {
+      text-decoration: underline;
+    }
+
+    &:disabled {
+      background: #626262;
+    }
+
+    &:hover {
+      background: #724fd8;
+      transform: scale(0.95);
```

Comment:
The hover state does not have any change other than background color. Please change the styles accordingly.